server/constants/vars.js

```diff
@@ -62,3 +64,49 @@ export const ALL_SUBJECTS_MONGO_PROJECTION = {
   _id: 0,
   subject: 1,
 }
+
+export const DEFAULT_CONFIG_AT_SIGNUP = (uid) => ({
+  owner: uid,
+  config: defaultUserConfig,
+  name: 'default',
+  type: 'matrix',
+  readers: [uid],
+  created: new Date().toUTCString(),
+})
```

Comment:
should this go in a config model?

```suggestion
const ConfigModel = {
  default: (overrides = {}) => ({
    config: defaultUserConfig,
    name: 'default',
    type: 'matrix',
    created: new Date().toUTCString(),
    ...overrides,
  }),
})

export default ConfigModel
```

Reply:
[de48040](https://github.com/AMP-SCZ/dpdash/pull/293/commits/de48040cb5b39f723d4ad1877be815b064027992)